.vsts-dnceng.yml

```diff
@@ -1,7 +1,7 @@
 phases:
 - phase: Windows_Desktop_Unit_Tests
   queue:
-    name: Helix
+    name: dnceng-linux-external-temp-staging
```

Comment:
These are Windows jobs and the new pool is a linux one. Seems like something is wrong here.

Reply:
hmm... good point.  copy-paste error